cores/esp8266/uart.c

```diff
@@ -72,6 +72,15 @@ struct uart_
     struct uart_rx_buffer_ * rx_buffer;
 };
 
+struct uartIsrContext_t
+{
+    uartInterruptHandler callback;
+    void* arg;
+};
+
+// NOTE: GCC will generated an invalid warning for the following line
```

Comment:
This is causing the build failures. I will be wrapping it with a pragma ignore like...
```
#pragma GCC diagnostic push
#pragma GCC diagnostic ignored "-Wmissing-braces"
    static volatile struct uartIsrContext_t s_uartInterruptContext[2] = { 0 };
#pragma GCC diagnostic pop
```